devtools/conda-envs/test.yaml

```
@@ -5,7 +5,7 @@ dependencies:
   # Core
   - python
   - cython =0.29.36
-  - numpy
+  - numpy>=2
```

```suggestion
  - numpy=2
```
I doubt v3 will drop anytime soon, but we probably wouldn't want that pulled in 😅